bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java

```diff
@@ -63,9 +63,11 @@
  */
 public class TestClusterAPI {
 
-  private static final int MAX_WAIT_SECONDS = 20;
-  private static final String TEST_CLUSTER_ID = "test-cluster-api";
+  public static final int MAX_WAIT_SECONDS = 20;
+  public static final String TEST_CLUSTER_ID = "test-cluster-api";
   public static final byte[] COLUMN_FAMILY = Bytes.toBytes("test_family");
```

Comment:
I copied the IntegrationTest's conventions.  If we change the Cluster Test names, we should change IntegrationTest names as well.  I'll make it so.